### ISSUE TITLE
Adding clips from Youtube doesn't work

### ISSUE THREAD

**Issue by Haikkari** [markokajzer/discord-soundbot#100]
Adding clips from Youtube has been working well for now but few days ago it stopped working. Adding clips through Discord still works without issue but trying to add from Youtube does nothing and produces output below.

Bot: 2.1.0-dev
Node: 12.19.0
NPM: 6.14.8
OS: Debian 4.19.132-1
Yarn: 1.22.10

```
(node:569) UnhandledPromiseRejectionWarning: Error: Could not find player config
    at exports.getBasicInfo (/home/haikkari/discord-soundbot/node_modules/ytdl-core/lib/info.js:90:13)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Map.getOrSet (/home/haikkari/discord-soundbot/node_modules/ytdl-core/lib/cache.js:24:19)
    at async exports.getInfo (/home/haikkari/discord-soundbot/node_modules/ytdl-core/lib/info.js:215:14)
    at async Map.getOrSet (/home/haikkari/discord-soundbot/node_modules/ytdl-core/lib/cache.js:24:19)
(node:569) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:569) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by markokajzer** [comment]
Hey thanks so much for reaching out!

We do not regularly use the YouTube download feature so this slipped under the radar. I'll investigate and fix as soon as I can!

**Comment by markokajzer** [comment]
Should be fixed with 46d3cb729ec2066e2e87b4b38037a884b8286326.

Please try and let me know if something else is wrong :)
